packages/legacy-workbench/src/css/create-similarity-index.css

```diff
@@ -28,3 +28,28 @@ yasgui-component .CodeMirror {
 code {
     background-color: var(--gw-body-background);
 }
+
+.ontotext-yasgui-tooltip .tooltip-box {
```

Comment:
The `ontotext-yasgui-web-component.css` file is imported in the Yasgui component directive template. Move the styling there so it applies to all pages that use Yasgui.